tests/bson/bson-int64-serialization-001.phpt

```diff
@@ -7,14 +7,14 @@ MongoDB\BSON\Int64 serialization (Serializable interface)
 <?php
 
 $tests = [
-    unserialize('C:18:"MongoDB\BSON\Int64":47:{a:1:{s:7:"integer";s:19:"9223372036854775807";}}'),
-    unserialize('C:18:"MongoDB\BSON\Int64":48:{a:1:{s:7:"integer";s:20:"-9223372036854775808";}}'),
-    unserialize('C:18:"MongoDB\BSON\Int64":28:{a:1:{s:7:"integer";s:1:"0";}}'),
+    '9223372036854775807',
+    '-9223372036854775808',
+    0,
 ];
 
 foreach ($tests as $test) {
-    var_dump($test);
-    var_dump($s = serialize($test));
+    var_dump($int64 = new MongoDB\BSON\Int64($test));
```

Comment:
Consider constructing the Int64 object outside of the loop to be consistent with other tests such as `bson-int64-get_properties-001.phpt`.

Related: https://github.com/mongodb/mongo-php-driver/pull/1417#discussion_r1190723745

Reply:
Updated here and below.